docs/en/connector-v2/source/Easysearch.md

```diff
@@ -0,0 +1,198 @@
+# Easysearch
```

Comment:
Please refer to the new document style

https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/MySQL-CDC.md